### DISCUSSION TITLE
I want to display a deep directory structure on a single line.

### DISCUSSION THREAD

**Discussion by fuchami** [nvim-tree/nvim-tree.lua#2416] [Q&A] [answered]
If the project consists of very deep directories, like a project written in Java, the outlook is not good.
Is it possible to display a deep directory structure in one line like vscode?

NeoVim
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/27998202/81eb9192-94b3-40fe-8257-b93ada2c3421)

VScode
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/27998202/0426197c-f73b-43a5-a004-73622575873f)

**Comment by alex-courtis** [comment, marked as answer]
Please see `:help nvim-tree.renderer.group_empty`

**Reply by fuchami** [reply]
solved! thank you!
